src/ontology/uberon.Makefile

```diff
@@ -209,163 +219,213 @@ mirror-ehdaa2: | $(TMPDIR)
+	$(ROBOT) remove -i $< --base-iri $(URIBASE)/MmusDv_ \
+		        --axioms external --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+# Ditto for HsapDv
+imports/local-hsapdv.owl: mirror/hsapdv.owl
+	$(ROBOT) remove -i $< --base-iri $(URIBASE)/HsapDv_ \
+		        --axioms external --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+# For EHDAA2, we need to preserve the AEO classes as well
+imports/local-ehdaa2.owl: mirror/ehdaa2.owl
+	$(ROBOT) remove -i $< \
+		        --base-iri $(URIBASE)/EHDAA2_ --base-iri $(URIBASE)/AEO_ \
+		        --axioms external --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+# For the life stages ontology, by construction it includes classes from
+# all over the place, and we need to preserve most of them. We do not
+# preserve the FBdv and WBls classes however, since they are provided
+# by local-fbdv and local-wbls respectively.
+SSSO_PREFIXES = BtauDv DpseDv DsimDv GgalDv GgorDv HsapDv MdomDv MmulDv MmusDv OariDv PpanDv PtroDv RnorDv SscrDv ZFS
+imports/local-ssso.owl: mirror/ssso.owl
+	$(ROBOT) remove -i $< \
+		        $(foreach pfx,$(SSSO_PREFIXES),--base-iri $(URIBASE)/$(pfx)) \
+		        --axioms external --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+# For the following ontologies, in addition to removing axioms about
+# external entities we also need to replace some old-style properties.
+# ----------------------------------------
+# EMAPA also needs translation between TS-style stages to MmusDv stages
+imports/local-emapa.owl: mirror/emapa.owl $(IMPORTDIR)/map-properties.tsv
+	$(ROBOT) rename -i $< --mappings $(IMPORTDIR)/map-properties.tsv \
+		        --allow-missing-entities true --allow-duplicates true \
+		 rename --mappings $(IMPORTDIR)/map-mouse-stages.tsv \
+		        --allow-missing-entities true \
+		 remove --base-iri $(URIBASE)/EMAPA_ --axioms external \
+		        --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+imports/local-ma.owl: mirror/ma.owl $(IMPORTDIR)/map-properties.tsv
+	$(ROBOT) rename -i $< --mappings $(IMPORTDIR)/map-properties.tsv \
+		        --allow-missing-entities true --allow-duplicates true \
+		 remove --base-iri $(URIBASE)/MA_ --axioms external \
+		        --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+imports/local-xao.owl: mirror/xao.owl $(IMPORTDIR)/map-properties.tsv
+	$(ROBOT) rename -i $< --mappings $(IMPORTDIR)/map-properties.tsv \
+		        --allow-missing-entities true --allow-duplicates true \
+		 remove --base-iri $(URIBASE)/XAO_ --axioms external \
+		        --preserve-structure false --trim false \
+		 convert -f ofn -o $@
+
+# And for those ones, we also remove the imports. They were last
+# refreshed in 2016, we don't want to bring in old issues from old
+# versions of Uberon, CL, RO, etc.
+# ----------------------------------------
+imports/local-poro.owl: mirror/poro.owl
+	$(ROBOT) remove -i $< --select imports --trim false \
```

Comment:
Sidenote: if the presence of imports is a general issue, it would have been safe to add this clause to all `imports/local-%` rules (for futureproofing). I know this is overly cautious, so I am happy for the comment to be ignored.

Reply:
I know, my statement was about the unlikely possibility that someone starts adding imports to their ontology; adding this statement just makes sure that everything runs; not a biggiue

Reply:
I’d rather not add code and complexity to deal with hypothetical problems. _If_ we observe that another ontology is publishing a release product with import declarations, _then_ we’ll add code to deal with that.

Reply:
I know you dont mean that in an _absolute_ sense, but lets roll with it given the low probability of the issue happening.